src/annotator/sidebar.js

```diff
@@ -150,12 +150,15 @@ export default class Sidebar extends Host {
     this.renderFrame = requestAnimationFrame(() => {
       this.renderFrame = null;
 
-      if (this._gestureState.final !== this._gestureState.initial) {
+      if (
+        this._gestureState.final !== this._gestureState.initial &&
+        this.frame
```

Comment:
Strictly speaking I believe `this.frame` is non-null here since I don't think `this._gestureState.final` will ever be different from `this._gestureState.initial` if using a custom container for the sidebar rather than the default container (`this.frame`). However connection this is quite indirect so it seems more obvious to add an explicit check before referencing `this.frame` inside the body of the `if` statement.